### ISSUE TITLE
Changes made in wave lost

### ISSUE THREAD

**Issue by Monovox** [SubtitleEdit/subtitleedit#4208]
When I time the subs visually in the waveform window while the main window is in source view, then 
sometimes when switching back to list view or just clicking somewhere on the waveform, all changes I just made to the timings are reversed and lost. This happens very often, but so far I couldn't find any regularity in this behavior, nor replicate it at will. I went over all open issues and no one else mentions this, Is it just me?
Thank you

**Comment by niksedk** [comment]
Thx for the info :)
The text needs to be updated when subtitle is changed via waveform - hopefully fixed here: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.5.15/SubtitleEditBeta.zip
Does that work better for you?

**Comment by Monovox** [comment]
Thanks for your response,
I switched to the beta and report back more fully after checking, But the first thing I saw is that when in source view in main window and I move a sub by dragging it in the waveform, both source and list lose the time position and jump to start of file.
I did see that the timing in the list changes simultaneously and I believe this must solve the problem I wanted to report. 
Thank you again your efforts and for this outstanding program.
